.github/workflows/run-tests-rvv.yml

```
@@ -0,0 +1,54 @@
+#
+# Copyright 2020 - 2022 Free Software Foundation, Inc.
+#
+# This file is part of VOLK
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+#
+
+name: Run VOLK tests on different RVV configurations
+
+on: [push, pull_request]
+
+jobs:
+  Tests:
+    runs-on: ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v4
```

```suggestion
    - uses: actions/checkout@v4
      with:
        submodules: "recursive"
```
This would probably make the best use of github actions to use integrated features.

---

Since you mentioned "outdated file", how do you prefer I implement the edits?
Should I continue amending the commit and force pushing, or create multiple commits, or something else?

---

It's up to you. My personal preference is to have commits that tell a story.

The "outdated file" issue was a concurrency issue on the GitHub side of things. I wouldn't worry about it. It was just a funny coincidence.